kairos-test-utils/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
License field will be necessary after synchronizing base branch.

Reply:
done https://github.com/cspr-rad/kairos/pull/34/commits/4276b0b3b7d0937e4755d040a8fc8c1e22562521